e2e/cypress/integration/bot_accounts/helpers.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+export function createBotInteractive(team, username = `bot-${getRandomId()}`) {
+    // # Visit the integrations
+    cy.visit(`/${team.name}/integrations/bots`);
+
+    // # Click add bot
+    cy.get('#addBotAccount').click();
+
+    // # fill+submit form
```

Comment:
```suggestion
    // # Fill and submit form
```

Reply:
Thanks! Not paying attention much during my copy paste :)

Reply:
Done, thanks.